CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## Next Version
 
 ### Bugs fixed
-- Update postgresql to fix CVE-2024-1597
+- Update Vert.x to 4.4.9 to fix CVE-2024-1023
+- Update Postgresql JDBC driver to fix CVE-2024-1597
```

Comment:
Don't see the version bump for the postgresql driver, was it already done previously?

Reply:
Its just the wording change. From `Update postgresql to fix CVE-2024-1597` to `Update Postgresql JDBC driver to fix CVE-2024-1597`.

Reply:
I see now, it's just updating the message